src/main/java/workflow/Keys.java

```diff
@@ -0,0 +1,23 @@
+package workflow;
+
+public class Keys {
```

Comment:
A file to hold constants as keys. we can only programatically define so much, this is how we are binding fronted elements to a black box of context in a concise manner. similarly, a set of command line operations could each be bound to the context using these keys in a CLI implementation.